app/views/admins/_stats.haml

```diff
@@ -0,0 +1,60 @@
+- content_for :head do
+  = stylesheet_link_tag :admin
+
+%h1= t(".usage_statistic")
+
+.row
+  = form_tag("/admins/stats", method: "get", class: "form-inline segment-selection") do
+    .col-sm-2.col-sm-push-8
+      %select.form-control{name: "range"}
+        %option{value: "daily", selected: ("selected" if params[:range] == "daily")}
```

Comment:
I'll do it. My point here is I usually don't refactor code I don't understand, I don't want to break stuff or have any side effects. So as a beginner I prefer to be careful.